i18n_generate.mll

```diff
@@ -118,50 +132,87 @@ let print_header fmt ?primary_module ~default_language () =
   in
   Format.pp_print_string fmt @@
   default_lang ^
-   "let%server _language_ = " ^ server_language_reference ^ "\n\
-   let%server get_language () = Eliom_reference.Volatile.get _language_\n\
-   let%server set_language language = \n\
-   Eliom_reference.Volatile.set _language_ language\n\
-   \n\
-   let%client _language_ = " ^ client_language_reference ^ "\n\
-   let%client get_language () = !_language_\n\
-   let%client set_language language = _language_ := language\n\
+  "let%server _language_ = " ^ server_language_reference ^ "\n\
+  let%server get_language () = Eliom_reference.Volatile.get _language_\n\
+  let%server set_language language = \n\
+  Eliom_reference.Volatile.set _language_ language\n\
+  \n\
+  let%client _language_ = " ^ client_language_reference ^ "\n\
+  let%client get_language () = !_language_\n\
+  let%client set_language language = _language_ := language\n\
+  \n\
+  let%shared txt = Eliom_content.Html.F.txt\n\
+  "
+
+let print_header fmt ?primary_module ~default_language () =
+  let default_lang =
+    match primary_module with
+    | None -> "let default_language = " ^ default_language ^ "\n"
+    | Some module_name -> ""
+  in
+  Format.pp_print_string fmt @@
+  default_lang ^
+  "let _language_ = default_language \n\
```

Comment:
is this used?

Reply:
I copied the Eliom version and removed all Eliom mentions. This part is also present in Eliom part, so I think it's used.